CONTRIBUTING.md

```diff
@@ -161,43 +165,53 @@ Once your changes and tests are ready to submit for review:
     one of the reviewers who reviewed your PR, but it need not be. The
     sponsor will issue the `/sponsor` command after you issue `/integrate`
     once they are satisfied that the review criteria have been met.
-
-6. Resolve the JBS bug as "Fixed"
-
-    There is currently no automation for resolving JBS bugs, although
-    a future Skara improvement will automate this. Until then,
-    the Committer who integrated or sponsored the fix is responsible for
-    resolving the JBS issue. You do this with the "Resolve" action in JBS,
-    selecting "Fixed" as the resolution. You also need to add the commit
-    notification message (minus the list of modified files) as a comment.
-    This includes the URL of the commit. For example:
-
-    ```
-    Changeset: 1de25a49
-    Author:    Kevin Rushforth <kcr@openjdk.org>
-    Date:      2019-09-23 08:15:36 +7000
-    URL:       https://git.openjdk.java.net/jfx/commit/1de25a49
-
-    8231126: libxslt.md has incorrect version string
-
-    Reviewed-by: ghb
-    ```
-
-
-Contributing to the OpenJFX codebase
+    The Skara bot will automatically resolve the JBS issue as "Fixed"
+    so there is no need to manually resolve it.
+
+
+New features / API additions
+----------------------------
+
+Adding a new feature to OpenJFX requires us to consider what it means to support that API forever; we take compatibility seriously.
+The main idea is to think in terms of "stewardship" when evolving the JavaFX API. This will require significant effort on your part.
```

Comment:
@johanvos to address your point about maintenance, what do you think about adding a sentence here? Something like:

This will require significant effort on your part. __It begins before you submit a pull request for review, and continues after the new feature is integrated.__

We can follow this up with more detail in the code review guidelines.

Reply:
That's a good sentence. It clearly and concisely tells what it is expected.